doc/integration.rst

```diff
@@ -93,10 +93,9 @@ Parameters
 +------------------------------+-------------------+------------+-----------------------------------------------------------------+
 | ``RV32M``                    | bit               | 1          | M(ultiply) extension enable                                     |
 +------------------------------+-------------------+------------+-----------------------------------------------------------------+
-| ``RV32B``                    | ibex_pkg::rv32b_e | RV32BNone  | *EXPERIMENTAL* - B(itmanipulation) extension select:            |
+| ``RV32B``                    | ibex_pkg::rv32b_e | RV32BNone  | B(itmanipulation) extension select:                             |
 |                              |                   |            | "RV32BNone": No B-extension                                     |
-|                              |                   |            | "RV32BBalanced": Sub-extensions Zbb, Zbs, Zbf and               |
-|                              |                   |            | Zbt                                                             |
+|                              |                   |            | "RV32BBalanced": Sub-extensions Zbb, Zbs, Zbf and Zbt           |
 |                              |                   |            | "RV32Full": All sub-extensions                                  |
```

Comment:
Please prefix the enum names with `ibex_pkg` as well.

Reply:
Done.